docs/one-way-objects.md

```diff
@@ -0,0 +1,47 @@
+@function can-stache-bindings.oneWayObjects One-way binding objects
+@parent can-stache-bindings.notes 0
+
+@body
+
+## One Way Binding for Objects
+
+`{child-prop}="key"` ([can-stache-bindings.toChild one-way to child]) or `{^child-prop}="key"` ([can-stache-bindings.toParent one-way to parent]) is used to pass values from the current scope to a component or vice versa, respectively.
+
+Generally, this binding only observes changes in one direction, but when [can-stache.key] is an object (POJO, DefineMap, etc), changes to the object itself are two-way; the one-way only applies to the object itself, not to changes to its properties.
```

Comment:
this sentence is hard for me to follow.  

> this binding

Which binding?  One way binding for object, one-way to child, or one-way to parent?

> changes to the object itself are two-way

what does this mean?  Does this mean that the object can be changed?